.circleci/config.yml

```diff
@@ -103,6 +103,8 @@ jobs:
       - run:
           name: Run pa11y-ci desktop
           command: npm run start-detached && npm run pa11y-ci:sitemap
+    # The resource_class feature allows configuring CPU and RAM resources for each job. Different resource classes are available for different executors. https://circleci.com/docs/2.0/configuration-reference/#resourceclass
+    resource_class: large
```

Comment:
Note that this commit was automatically generated by CircleCI. I don't care about this comment in the config, but I'm happy to delete it if it's distracting